gadget/gadgettest/examples.go

```diff
@@ -956,6 +1016,93 @@ var VMSystemVolumeDiskMapping = &disks.MockDiskMapping{
 	},
 }
 
+var VMSystemVolumeClassicDiskMapping = &disks.MockDiskMapping{
```

Comment:
this doesn't seem to be used atm?

Reply:
Right, this must be a left-over. Removed now.